libraries/AP_GPS/AP_GPS_GSOF.cpp

```diff
@@ -221,29 +222,19 @@ AP_GPS_GSOF::SwapFloat(uint8_t* src, uint32_t pos)
 uint32_t
 AP_GPS_GSOF::SwapUint32(uint8_t* src, uint32_t pos)
 {
-    union {
-        uint32_t u;
-        char bytes[sizeof(uint32_t)];
-    } uint32u;
-    uint32u.bytes[0] = src[pos + 3];
-    uint32u.bytes[1] = src[pos + 2];
-    uint32u.bytes[2] = src[pos + 1];
-    uint32u.bytes[3] = src[pos + 0];
-
-    return uint32u.u;
+    // Could also try using std::copy_backward
```

Comment:
Sorry, we don't want this comment.  We do not want to encourage people to use the standard library in ArduPilot ATM.  It's non-trivial to use it and there's going to be a support load.

Reply:
fcafc31b145ded2dc3559adfc33ac7cf45222757